src/framework/theme/components/calendar/calendar.component.ts

```diff
@@ -20,6 +20,20 @@ import { NbCalendarCell, NbCalendarSize, NbCalendarViewMode } from '../calendar-
  * Basic usage example
  * @stacked-example(Showcase, calendar/calendar-showcase.component)
  *
+ * ### Installation
+ *
+ * Import `NbCalendarRangeModule` to your feature module.
```

Comment:
And here not `NbCalendarRangeModule` but `NbCalendarModule`

Reply:
Maybe we can merge calendar and calendar range modules? Do we really need this separation?

Reply:
let's keep both for now